runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -2023,26 +2265,49 @@ J9::Z::PrivateLinkage::buildVirtualDispatch(TR::Node * callNode, TR::RegisterDep
       if (numInterfaceCallCacheSlots == 0 )
          {
          //Disabled interface call caching
+         TR::Instruction * cursor = NULL;
          TR::LabelSymbol * hitLabel = generateLabelSymbol(cg());
          TR::LabelSymbol * snippetLabel = generateLabelSymbol(cg());
+         TR::LabelSymbol * returnLocationLabel = generateLabelSymbol(cg());
+         TR::LabelSymbol * paramSetupDummyLabel = generateLabelSymbol(cg());
+         TR::Register * RegRA = dependencies->searchPostConditionRegister(getReturnAddressRegister());
+         TR::Register * RegEP = dependencies->searchPostConditionRegister(getEntryPointRegister());
+         TR::Register * RegThis = dependencies->searchPreConditionRegister(TR::RealRegister::GPR1);
```

Comment:
Can you comment on why RegThis is searched from deps and then assigned to `assignAny` in postDeps ?